templates/CMakeLists.txt.template

```diff
@@ -264,13 +264,18 @@
     endforeach()
   endfunction()
 
-  add_custom_target(plugins
-    DEPENDS
+  set(_gRPC_PLUGIN_LIST)
   % for tgt in targets:
   % if tgt.build == 'protoc':
-    ${tgt.name}
+  option(gRPC_BUILD_${tgt.name.upper()} "Build ${tgt.name}" ON)
```

Comment:
nit: worth adding a comment? It's hard to guess for someone who doesn't already know.

Reply:
Add a comment in the code? Or to the documentation? Where?

Reply:
Yes, I agree, I will add a comment explaining it.